docs/modules/services/pages/geocoding.adoc

```diff
@@ -1,34 +1,35 @@
-# Geocoding
+= Geocoding
```

Comment:
Maybe this section it could be reworked as due to PR #6340 made changes and restructured this section, making disappear geocoding.md in favor of [maps.md](https://github.com/decidim/decidim/blob/develop/docs/services/maps.md)

Reply:
Same as my [other comment](https://github.com/decidim/decidim/pull/6526#discussion_r500372779), I need to manually merge last commits

Reply:
Fixed that: https://github.com/decidim/decidim/pull/6526#issuecomment-704392634